env.juno.sh

```diff
@@ -5,6 +5,8 @@
 # set -eu # NOTE: do not use this because it can propagate into cwl-tool subprocesses and make them break unneccessarily
 arg="${1:-None}"
 
+. /etc/profile.d/modules.sh
```

Comment:
@nikhil @allanbolipata we need to wrap this in a `[ -e ] && ... : ...` statement I think since the tests might not be running from a location with access to modules.sh